CHANGELOG.md

```diff
@@ -11,21 +11,44 @@ Branch: `nilrt/master/hardknott`
 
 ### NILRT
 
+#### Added
+- Added [Pyrex](https://github.com/garmin/pyrex) usage for build containers to manage dependencies and versions.
+
 #### Changed
 - Rebased OE layer submodules from the OE/`sumo` release stable branches, to the OE/`hardknott` branches - where available.
   - A full list of the upstream changes between OE/sumo (yocto 2.5) and OE/hardknott (yocto 3.3) can be found in the Yocto project documentation's [Migration Guide](https://docs.yoctoproject.org/migration-guides/index.html).
 - Upgraded the bitbake submodule to bitbake `1.50.2`.
 - [Upgraded](https://github.com/ni/nilrt/pull/73) the nilrt-build dockerfile to a debian 10 base.
   - Added a more user-friendly way to enter the docker build container using `docker-compose`.
 
+#### Fixed
+- [Fixed](https://github.com/ni/meta-nilrt/pull/308) an issue where SSH sessions were not properly closed on reboot or shutdown.
+
+#### Removed
+- Removed the following OpenEmbedded Layers:
+  - [meta-ivi](https://github.com/ni/nilrt/pull/119).
+  - [meta-java](https://github.com/ni/nilrt/pull/120).
+  - [meta-measured](https://github.com/ni/nilrt/pull/127).
+  - [meta-mingw](https://github.com/ni/nilrt/pull/124).
+  - [meta-mono](https://github.com/ni/nilrt/pull/123).
+
 
 ### meta-nilrt
 
+#### Added
+ - [Added](https://github.com/ni/meta-nilrt/pull/277) `nilrt-recovery-image` to build safe mode for hardknott targets.
+
 #### Changed
 - `gcc` has been upgraded to version `10.2`.
 - `openssl` has been upgraded to `1.1.1k`.
 - `python2` support has been totally deprecated in favor of `python3`.
+- `opkg` has been upgraded to `0.6.0`.
```

Comment:
I'm not going to merge the `0.6.0` PRs into the 22.5 release. So either set this to `0.5.0`, if we upgraded that version since the last CHANGELOG, or remove the line entirely.